plugins/en/qml/Keyboard_en_tablet.qml

```diff
@@ -90,8 +92,8 @@ KeyPad {
             CharKey { label: "b"; shifted: "B"; }
             CharKey { label: "n"; shifted: "N"; extended: ["ñ"]; extendedShifted: ["Ñ"] }
             CharKey { label: "m"; shifted: "M"; }
-            CharKey { label: ","; shifted: "/" }
-            CharKey { label: "."; shifted: "?" }
+            CharKey { imgNormal: UI.imageGreyKey[formFactor]; imgPressed: UI.imageGreyKeyPressed[formFactor]; label: ","; shifted: "/"; /*extended:"/";*/}
```

Comment:
I don't think "formFactor" exists in this context (maybe it works in practice because the variable has been inherited). Anyway, in this file, we are necessarily in the "tablet" form factor.
Same remark for the following lines.